cpp/include/cugraph_c/centrality_algorithms.h

```diff
@@ -319,23 +314,18 @@ void cugraph_edge_centrality_result_free(cugraph_edge_centrality_result_t* resul
 /**
  * @brief     Compute edge betweenness centrality
  *
- * Edge betweenness can be computed exactly by specifying num_vertices as 0 and
- * vertex_list as NULL.  This will compute betweenness centrality by doing a
- * traversal from every vertex and counting the frequency that a edge appears on
- * a shortest path.
- *
- * Approximate betweenness can be computed either by specifying num_vertices > 0,
- * which will randomly pick the specified number of seeds; or by specifying a
- * list of vertices that should be used as seeds for the BFS.
+ * Edge betweenness can be computed exactly by specifying vertex_list as NULL.  This will compute
+ * betweenness centrality by doing a traversal from every vertex and counting the frequency that a
+ * edge appears on a shortest path.
  *
- * Specifying both num_vertices > 0 and vertex_list as non-null will result in an
- * error as the request is ambiguous.
+ * Approximate betweenness can be computed specifying a list of vertices that should be
+ * used as seeds for the traversals.  Note that the function cugraph_select_random_vertices can be
+ * used to create a list of seeds.
  *
  * @param [in]  handle             Handle for accessing resources
  * @param [in]  graph              Pointer to graph
- * @param [in]  num_vertices       Number of vertices to randomly sample
  * @param [in]  vertex_list        Optionally specify a device array containing a list of vertices
- *                                 to use as seeds for BFS
+ *                                 to use as seeds for traversals
```

Comment:
Better match the documentation in vertex betweenness centrality?

In the above, we're saying `to use as seeds for betweenness approximation`.

Users may be confused that `vertex_list` may have different roles in vertex & edge betweenness centrality.